bin/main.rs

```diff
@@ -75,6 +102,47 @@ struct Args {
         env = "WADM_MAX_JOBS"
     )]
     max_jobs: usize,
+
+    /// URL of the Nats Jetstream KV that will be used for storage
+    #[arg(long = "storage-nkv-url", env = "STORAGE_NATSKV_URL")]
+    storage_nkv_url: Option<String>,
+
+    /// Optional prefix for the server tate that will be stored.
+    /// this prefix will be prepended to the bucket name of the lattice
+    /// (ex. <prefix>_lattice_<lattice id>).
+    #[arg(
+        long = "storage-nkv-lattice-bucket-prefix",
+        env = "STORAGE_NATSKV_LATTICE_BUCKET_PREFIX"
+    )]
+    storage_nkv_lattice_bucket_prefix: Option<String>,
+
+    /// (Optional) NATS NKey authentication
+    #[arg(
+        long = "storage-nkv-nats-auth-nkey",
+        env = "STORAGE_NATSKV_NATS_AUTH_NKEY"
+    )]
+    storage_nkv_nats_auth_nkey: Option<String>,
+
+    /// (Optional) NATS credential file to use when authenticating
+    #[arg(
+        long = "storage-nkv-nats-auth-creds-file",
+        env = "STORAGE_NATSKV_NATS_AUTH_CREDS_FILE"
+    )]
+    storage_nkv_nats_auth_creds_file: Option<String>,
```

Comment:
Can you make this mutually exclusive with nkey and seed? It'll make it clear to anyone who tries to specify both

Reply:
- [ ] make auth creds mutually exclusive with NATS JWT seed